friendsofsymfony/user-bundle/2.1/config/packages/fos_user.yaml

```diff
@@ -0,0 +1,7 @@
+fos_user:
+    db_driver: orm
+    firewall_name: main
+    user_class: App\Entity\User
+    from_email:
+        address: "address"
+        sender_name: "name"
```

Comment:
@tattali FOSUserBundle can be used without `templating` service. Add this to use twig directly:

```yaml
fos_user:
    service:
        mailer: fos_user.mailer.twig_swift
```

It can be used even without `mailer` service, if a bug will be fixed (PR is not accepted/merged yet). See https://github.com/FriendsOfSymfony/FOSUserBundle/issues/2704

Reply:
Ok sorry when I did the search I did not see your recipe. If you want we can use yours that looks more accomplished.

Reply:
There is no my recipe as there is no v2.1 release yet =) And it is not possible to create recipe right now, because even with new `fos_user.service.mailer` parameter, FOSUB will require doctrine-bundle for `fos_user.db_driver: orm`. I've created PR https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2708 but they ignore me =)